internal/worker.go

```diff
@@ -110,23 +111,30 @@ type (
 		// cadence-server to retrieve decision tasks. If FeatureFlags.PollerAutoScalerEnabled is set to true,
 		// changing this value will NOT affect the rate at which the worker is able to consume tasks from a task list.
 		// Default value is 2
+		// Deprecated: Use AutoScalerOptions instead.
 		MinConcurrentDecisionTaskPollers int
 
 		// optional: Sets the interval of poller autoscaling, between which poller autoscaler changes the poller count
 		// based on poll result. It takes effect if FeatureFlags.PollerAutoScalerEnabled is set to true.
 		// Default value is 1 min
+		// Deprecated: Use AutoScalerOptions instead.
 		PollerAutoScalerCooldown time.Duration
 
 		// optional: Sets the target utilization rate between [0,1].
 		// Utilization Rate = pollResultWithTask / (pollResultWithTask + pollResultWithNoTask)
 		// It takes effect if FeatureFlags.PollerAutoScalerEnabled is set to true.
 		// Default value is 0.6
+		// Deprecated: not used any more
 		PollerAutoScalerTargetUtilization float64
 
 		// optional: Sets whether to start dry run mode of autoscaler.
 		// Default value is false
+		// Deprecated: not used any more
 		PollerAutoScalerDryRun bool
 
+		// optional: Sets the options for poller autoscaler
+		AutoScalerOptions AutoScalerOptions
```

Comment:
I'm kinda curious if we'll want to split this into decision/activity options eventually..... but seems probably fine for now anyway.  and simpler to config if they're shared.

Reply:
My thought was to simplify things. I don't think it needs tuning in the first place so a single config should be good enough